### PR TITLE
CCS-4062: added tests to get id of created product and version

### PR DESCRIPTION
**Pull request by pbadak**

### CODE REVIEW

**Review by anarang — COMMENTED** [review]
@pbadak please create a separate test file with covers the respective endpoints, also keep into consideration the tear down step to delete the new product and version created just in case you create one.

**Comment by pbadak** [comment]
review comments incorporated @anarang @noopurmath3

File: suites/test_product.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        path_to_product_id = self.path_to_product + "/versions/" + constants.product_version + ".json"
+        product_version_id_req = requests.get(path_to_product_id)
+        check_that("%s API status code" % path_to_product_id, product_version_id_req.status_code, equal_to(200))
+        product_version_id = product_version_id_req.json()
+        check_that("Product version uuid", product_version_id["jcr:uuid"], is_not_none())
+        check_that("sling:resourceType", product_version_id["sling:resourceType"], equal_to("pantheon/productVersion"))
+        check_that("jcr:primaryType", product_version_id["jcr:primaryType"], equal_to("pant:productVersion"))
+        check_that("product version", product_version_id["name"], equal_to(constants.product_version))
+
+    @lcc.test("Delete above created product")
```
Comment (anarang) [inline suites/test_product.py:65]: @pbadak this should not be a test, but a tear down step for the test case, the product node should be deleted irrespective that the above tests are successful or not. In addition, add a try catch that the delete should not fail or give an error if the product does not exist given that the above tests failed for some reason.

**Review by anarang — CHANGES_REQUESTED** [review]
@pbadak added a comment for you to address, rest it looks fine.

**Review by noopurmath3 — APPROVED** [review]

File: fixtures/fixture.py [hunk 2]
```diff
@@ -139,9 +148,11 @@ def setup_test_products():
     time.sleep(5)
     path_to_product_id = path_to_product_node + ".2.json"
     product_version_id_req = requests.get(path_to_product_id)
+    check_that("%s API works as expected" % path_to_product_id, response.status_code, any_of(equal_to(201), equal_to(200)))
```
[the remark this answers is not in the document]
Comment (pbadak) [reply fixtures/fixture.py]: we are not making any changes to fixture file anymore.

File: suites/test_product.py (hunk 1 above)
Comment (pbadak) [reply suites/test_product.py]: Thanks for your review @anarang . Made changes as per comment .

**Pull request merged**